app/gui/CliQuitStreamSegue.qml

```diff
@@ -0,0 +1,70 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.2
+import QtQuick.Dialogs 1.2
+
+import ComputerManager 1.0
+import Session 1.0
+
+Item {
+    anchors.fill: parent
+
+    function onSearchingComputer() {
+        stageLabel.text = "Establishing connection to PC..."
+    }
+
+    function onQuitingApp() {
+        stageLabel.text = "Quiting app..."
```

Comment:
`Quitting` is misspelled here and in a few function names

Reply:
Ok